### ISSUE TITLE
meal_workout 

### ISSUE THREAD

**Issue by suryanevcodes** [suryanevcodes/healthfyme-#7]
Implement the functionality for generating exercise plans based on user BMI categories. Recommend exercise routines, duration, and intensity levels.(3 plans)

**Comment by suryanevcodes** [comment]
add 3 simple plans so that it in a tabular form dont use sql fro this just make it with sout

**Comment by MoulaNeverCodes** [comment]
Yeah i have added 3 plans for meal and as well as workout. Check and reflect if any changes are required.

**Comment by MoulaNeverCodes** [comment]
Verified the code but the integration part is left which has to be done by End of Day. Happy Integration ☺️  @suryanevcodes @123anjanas @Prasanna-neo

**Comment by MoulaNeverCodes** [comment]
completed and verified push it to the main branch and don't do any updates now
